catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -52,6 +52,7 @@
 	public void onCreate() {
 		super.onCreate();
 		Log.d(TAG, "CatroidApplication onCreate");
+		Log.d(TAG, "git commit info: " + BuildConfig.GIT_COMMIT_INFO);
```

Comment:
The log looks nicer if the commit info is logged after `CatroidApplication onCreate`.
```
10-10 16:27:36.178 29269-29269/org.catrobat.catroid D/CatroidApplication: CatroidApplication onCreate
    git commit info: branch log-git-commit-info @cf0c45c0d
```